vars/Rocky_8.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: MIT
+---
+# Put internal variables here with Rocky Linux 8 specific values.
+
+__nbde_client_packages:
+  - clevis
+  - clevis-dracut
+  - clevis-luks
+  - clevis-systemd
+  - iproute
+  - NetworkManager
+
+__nbde_client_initramfs_update_cmd: >
```

Comment:
Why use `>`? You can type `__nbde_client_initramfs_update_cmd: dracut -fv --regenerate-all`

Reply:
I literally just copied the RedHat_8.yml file, renamed it and reworded the comment at the top so that the vars would get pulled in when running against a Rocky Linux system. The Alma and CentOS vars files are also identical (outside the comment) to the RedHat ones, so it may even make sense to add a check in set_vars.yml for `{{ ansible_facts['os_family'] }}_{{ ansible_facts['distribution_major_version'] }}.yml` and quit it with the seperate files for each EL flavor. Not sure which would be prefered.

Reply:
Good points, I wasn't aware some of those other OSes still fell under the RedHat os family. Back to the original comment then, would it be better to move those to the single line as you suggested, or keep as is for consistency with the other RedHat-derived vars files?